get_data_for_figures.py

```diff
@@ -86,23 +92,25 @@ def get_random_samples_of_populations(mt, meta_ht, pops, n):
     selected_samples = set([])
     for pop in pops:
         random_samples = get_random_subset(meta_ht, n, pop)
-        selected_samples = selected_samples | set(random_samples) #what's the preferred syntax for adding onto a set?
-    mt = get_hardcalls_of_samples(mt, selected_samples)
+        selected_samples = selected_samples | set(
+            random_samples
+        )  # what's the preferred syntax for adding onto a set?
```

Comment:
I would go with Ben's suggestion
```suggestion
        selected_samples |= set(random_samples)
```